tests/bin/libc.h

```diff
@@ -73,3 +73,18 @@ int is_enarx() {
         default: return 0;
     }
 }
+
+ssize_t get_att(void *nonce, size_t nonce_len, void *buf, size_t buf_len, size_t *technology) {
+    ssize_t rax;
+    ssize_t tech;
+    register size_t r10 __asm__("r10") = buf_len;
+
+    asm(
+        "syscall"
+	: "=a" (rax), "=d" (tech)
+	: "a" (0xEA01), "D" (nonce), "S" (nonce_len), "d" (buf), "r" (r10)
+    );
```

Comment:
I recently learned that the `syscall` instruction clobbers `rcx` and (I think...) `r11`. That could be a source of the bug here.

Reply:
I have added fixes for the existing syscalls in #101.

Reply:
Good to know!